README.md

```diff
@@ -1,10 +1,83 @@
-### Running locally
+# nextjs_dev_folio
 
-```
-npm run dev
-```
+## About
+
+`nextjs_dev_folio` is a personal portfolio website built using Next.js. This project allows developers to showcase their skills, projects, and personal information in an organized and visually appealing manner. This repository is open for **Hactoberfest 2023!!**
+
+## Getting Started
+
+### Running Locally
+
+To run the project locally, follow these steps:
+
+1. Clone this repository to your local machine:
+
+   ```bash
+   git clone https://github.com/danger-ahead/nextjs_dev_folio.git
+   ```
+
+2. Navigate to the project directory:
+
+   ```bash
+   cd nextjs_dev_folio
+   ```
+
+3. Install the project dependencies:
+
+   ```bash
+   npm install
+   ```
+
+4. Start the development server:
+
+   ```bash
+   npm run dev
+   ```
+
+The application should now be accessible locally at `http://localhost:3000`.
 
 ### Requirements
 
-- To display your picture on the About page, upload your picture to [imgur](https://imgur.com/) and paste the link to the image in [data.json](https://github.com/danger-ahead/nextjs_dev_folio/blob/bcbe65c293f114242fcbb0585e397951b2f0dab6/public/data.json#L3).
-- Copy `.env.example` to `.env.local` and update the values.
+To customize the portfolio and add your personal information, follow these steps:
+
+1. Upload your picture to [imgur](https://imgur.com/).
+
+2. Copy the link to the image.
+
+3. Open the `public/data.json` file in your project and paste the image link at [data.json](https://github.com/danger-ahead/nextjs_dev_folio/blob/bcbe65c293f114242fcbb0585e397951b2f0dab6/public/data.json#L3).
```

Comment:
Also mention about the contents in the `static` directory.

Reply:
Means? Should i tell the readers to go through the static folder or add a section with the content that is in the static folder?

Reply:
The content that is in the static folder is about your experience so I m thinking to add a owner information section with your experience and description. If you want to add your linkedIn profile link then that can also be done!!

Reply:
Please tell me what changes you want related to static

Reply:
Yeah, so the contents in the portfolio, for example:
- about
- experience details
And some other stuff come from the mdx files in the static dir. Can you simply mention this? So that users know which directory to modify.

Reply:
ok

Reply:
## Owner Experience

- **Software Development Enthusiast**: I have a strong passion for software development and have been actively involved in various projects, including open-source contributions and personal endeavors.
- **Startup Collaborations**: I've had the privilege of working with startups, where I've gained a deep understanding of the significance of well-designed products and their impact on user experiences.
- **Clean Code Advocate**: I'm unapologetically a "clean code" enthusiast, believing that elegantly designed and well-structured code improves overall software quality and maintenance.
- **Open-Source Contributions**: I'm deeply engaged in the open-source community, making significant contributions to projects like Next.js, CircuitVerse Mobile App, Flutter Community, and Invoice Ninja Admin Portal.
- **Ongoing Learning**: Currently, I'm delving into the depths of Flutter and web development using JS-based frameworks, enjoying the process of learning about the intricacies of these technologies.


How's this? @danger-ahead

Reply:
Hey sorry. This is not what I meant. These are my personal portfolio details. 

Explain to the user that he/she needs to edit the mdx files in the static directory to change the contents of the portfolio.

Reply:
Oh okayy sorry . I will update that..